static/app/views/replays/detail/replayMetaData.tsx

```diff
@@ -107,21 +110,22 @@ const KeyMetrics = styled('div')`
 const KeyMetricData = styled('div')`
   color: ${p => p.theme.textColor};
   font-weight: normal;
-  display: grid;
-  grid-template-columns: repeat(2, max-content);
```

Comment:
When this is grid and `repeat(2, ...)` it requires that there be two items inside. This wasn't the case for the <ProjectBadge> item, and now not the case for the error count.